### DISCUSSION TITLE
Air conditioner

### DISCUSSION THREAD

**Discussion by andrazek** [gekkekoe/esphome-ecodan-hp#87] [Announcements]
I have closely followed instructions and installed one such device in my air conditioner MSZ-AP25VG. How to get it working? All the sensors show "unknown" state. Should I set something special under "Controls"?

**Comment by gekkekoe** [comment]
Hardware works, but you need different software for airco. Use something like https://github.com/geoffdavis/esphome-mitsubishiheatpump

**Reply by andrazek** [reply]
Thank you! Awesome project, by the way. Your instructions were super easy to follow, and I’ve learned a ton along the way. For my Ecodan heat pump, I’m pretty sure your software is the way to go.

For my aircon, though, I ended up using this one instead (looks like it gets updated more often): https://github.com/fonske/MitsubishiCN105ESPHome/blob/main/me_airco_atom_s3_lite.yaml

I can say it just works. Vertical vane control also works! The only thing I miss is energy consumption (which MELCloud had).

**Reply by gekkekoe** [reply]
Ah great, yeah forgot about that one, @fonske also made the proxy pcb
